src/components/RadioGroup/RadioGroup.spec.jsx

```diff
@@ -48,14 +48,6 @@ describe('RadioGroup', () => {
 			});
 		});
 
-		describe('onSelect', () => {
```

Comment:
this was not a good test

1. i don't think we should test that `defaultProps` works properly
2. this was testing the exact wrong thing, i.e. that `onSelect` was getting passed to the root `span`, which we shouldn't be doing, and which `omitProps` prevents